data/remnant missions.txt

```diff
@@ -60,27 +62,94 @@ mission "First Contact: Remnant"
 	
 	on decline
 		event "ember waste label"
+		set "remnant blood test pure"
 		log "Discovered a strange, isolated human community in an otherwise uninhabited region of space south of the Core. They call themselves the Remnant."
 		log "Factions" "Remnant" `Half a millennium ago, during the Alpha Wars, a group of humans fleeing the conflict and looking for a safer place to live discovered a wormhole into a region of space known as the Ember Waste. They built settlements there, and chose to call themselves the Remnant. Although they no longer fear the Alphas as much as they once did, they still have not rejoined human society.`
 			`Since their colonies were first formed, the Remnant's culture has changed drastically. They communicate in sign language, using their voices only for singing. Their food and architecture and much of their technology is strange, and much of it has been borrowed or stolen from nearby alien species.`
 	
 	npc kill
 		government "Remnant"
 		personality heroic coward disables plunders
-		fleet "Large Remnant" 2
+		fleet "Large Remnant"
 
 event "ember waste label"
 	galaxy "label waste"
 		sprite label/waste
 
+mission "Remnant: Second Contact"
+	name "Second chance at First Contact with the Remnant"
+	landing
+	invisible
+	source
+		government "Remnant"
+	description "A Remnant provides you with the opportunity to take a second run at first contact."
+	clearance "You can't understand the video feed of gestures, but your ship's computer receives a landing grid solution via antique protocols."
+	destination Viminal
+	waypoint Segesta
+	to offer
+		not "remnant blood test pure"
+		not "Remnant Second Chance Refused"
+	to fail
+		or
+			has "Remnant Second Chance Refused"
+			has "remnant blood test pure"
+	on enter Segesta
```

Comment:
Changed to Stercutus for the moment, but changing it so that it triggers in the first uninhabited Ember Waste system the player reaches would probably be better.